test/system/libs/helpers.bash

```diff
@@ -214,7 +213,7 @@ function _setup_docker_registry() {
 # Stop, removes and cleans after a locally hosted Docker registry
 function _clean_docker_registry() {
   # Stop Docker registry container
-  $PODMAN --root "${DOCKER_REG_ROOT}" stop --time 0 "${DOCKER_REG_NAME}"
+  $PODMAN --root "${DOCKER_REG_ROOT}" stop --time 0 "${DOCKER_REG_NAME}" || true
```

Comment:
What does this fix?

Reply:
It was supposed to force the program to continue and not fail even if the line failed, but didn't work out.